cvat/apps/dataset_manager/formats/registry.py

```diff
@@ -52,6 +52,16 @@ def __call__(self, *args, **kwargs):
     return target
 
 EXPORT_FORMATS = {}
+
+def format_for(format, mode):
+    format_name = format
+    if mode == "annotation":
+        format_name = "CVAT for images 1.1"
+    elif format not in EXPORT_FORMATS:
+        format_name = "CVAT for video 1.1"
+    return format_name
```

Comment:
Could you explain the logic behind this function?

Reply:
This is an additional check if we can use selected git format with this task mode and if this format is presented in EXPORT_FORMATS. Also you may see this logic here: cvat/apps/dataset_repo/dataset_repo.py line 268 (in this branch moved to registry.py)